core/src/main/java/pl/agh/edu/ui/component/bankOffer/BankOffer.java

```diff
@@ -0,0 +1,71 @@
+package pl.agh.edu.ui.component.bankOffer;
+
+import static com.badlogic.gdx.utils.Align.center;
+import static com.badlogic.gdx.utils.Align.left;
+import static com.badlogic.gdx.utils.Align.right;
+import static pl.agh.edu.ui.utils.SkinColor.GRAY;
+import static pl.agh.edu.ui.utils.SkinFont.BODY2;
+import static pl.agh.edu.ui.utils.SkinFont.H3;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+
+import pl.agh.edu.GdxGame;
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.data.type.BankData;
+import pl.agh.edu.engine.bank.BankAccount;
+import pl.agh.edu.ui.component.ClickableTable;
+import pl.agh.edu.ui.component.label.LanguageLabel;
+import pl.agh.edu.ui.utils.SkinColor;
+import pl.agh.edu.utils.LanguageString;
+
+public class BankOffer extends ClickableTable {
+
+	public final BankData bankData;
+	BankAccount bankAccount = ((GdxGame) Gdx.app.getApplicationListener()).engine.bankAccountHandler.account;
+
+	public BankOffer(BankData bankData) {
+		super();
+		this.bankData = bankData;
+
+		String whiteFont = BODY2.getWhiteVariantName();
```

Comment:
nawet jeżeli jest to wartość stała wyrzucił bym do stylów żeby nie nawalać tu kodu